howso/hierarchy.amlg

```diff
@@ -356,14 +232,55 @@
 			;name to load (without extension)
 			filename ""
 			;{type ["string" "list"] values "string"}
```

Comment:
Seems like this type should just be "list"?

Reply:
Same goes for some of the other methods, all should accept the same type for path.